GUI/GPTMDTaskWindow.xaml

```diff
@@ -418,10 +430,27 @@
                                 <ColumnDefinition Width="1*"/>
                             </Grid.ColumnDefinitions>
                             <Grid.RowDefinitions>
+                                <RowDefinition Height="25"/>
                                 <RowDefinition Height="25"/>
                                 <RowDefinition Height="1*"/>
                             </Grid.RowDefinitions>
                             <StackPanel Grid.Column="0" Grid.Row="0" Orientation="Horizontal">
+                                <Label x:Name="MaxModsPerNotchLabel" Content="Max Mods Per Notch" />
+                                <TextBox x:Name="MaxModsPerNotchTextBox" PreviewTextInput="CheckIfNumber" Width="45" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="500" IsEnabled="False">
```

Comment:
it seems kind of weird to have a textbox that's not enabled; can you put the extra info about combos in the tooltip for "Max Mods Per Peptide Before GPTMD"?